### ISSUE TITLE
SA1503 (CurlyBracketsMustNotBeOmitted) ignores do/while statements

### ISSUE THREAD

**Issue by sharwell** [DotNetAnalyzers/StyleCopAnalyzers#859]
Currently SA1503 does not analyze do/while statements. For example, the following should report SA1503:

```csharp
do
    Method();
while (false);
```
